config/webpack.config.js

```diff
@@ -417,7 +417,7 @@ module.exports = function (webpackEnv) {
         maxChunks: 1
       }),
       new MonacoWebpackPlugin({
-        languages: ['json', 'javascript', 'java', 'csharp', 'html', 'powershell', 'go']
+        languages: ['json', 'javascript', 'java', 'csharp', 'html', 'powershell', 'go', 'python']
```

Comment:
@shemogumbe we are now ordering the code snippets tab by alphabet.

Reply:
Hey @adhiambovivian. The ordering here does not affect the ordering on the UI